moto/ec2/responses/vpcs.py

```diff
@@ -486,3 +522,65 @@ def describe_vpc_endpoint_services(self):
         </item>
     </serviceDetailSet>
 </DescribeVpcEndpointServicesResponse>"""
+
+DESCRIBE_VPC_ENDPOINT_RESPONSE = """<DescribeVpcEndpointsResponse xmlns="http://ec2.amazonaws.com/doc/2016-11-15/">
+    <requestId>19a9ff46-7df6-49b8-9726-3df27527089d</requestId>
+    <vpcEndpointSet>
+        <item>
+            {% for vpc_end_point in vpc_end_points %}
+                <vpcEndpointId>{{ vpc_end_point.id }}</vpcEndpointId>
+                <vpcEndpointType>{{ vpc_end_point.type }}</vpcEndpointType>
+                <vpcId>{{ vpc_end_point.vpc_id }}</vpcId>
+                <serviceName>{{ vpc_end_point.service_name }}</serviceName>
+                {% if vpc_end_point.policy_document  %}
+                    <policyDocument>{{ vpc_end_point.policy_document }}</policyDocument>
+                {% endif %}
+                {% if vpc_end_point.route_table_ids  %}
+                    <routeTableIdSet>
+                        {% for route_table_id in vpc_end_point.route_table_ids %}
+                                <item>{{ route_table_id }}</item>
+                        {% endfor %}
+                    </routeTableIdSet>
+                {% endif %}
+                {% if vpc_end_point.subnet_ids  %}
+                    <subnetIdSet>
+                        {% for subnet_id in vpc_end_point.subnet_ids %}
+                            <item>{{ subnet_id }}</item>
+                        {% endfor %}
+                    </subnetIdSet>
+                {% endif %}
+                {% if vpc_end_point.network_interface_ids  %}
+                    <networkInterfaceIdSet>
+                        {% for network_interface_id in vpc_end_point.network_interface_ids %}
+                            <item>{{ network_interface_id }}</item>
+                        {% endfor %}
+                    </networkInterfaceIdSet>
+                {% endif %}
+                {% if vpc_end_point.dns_entries  %}
+                    <dnsEntrySet>
+                        {% for dns_entry in vpc_end_point.dns_entries %}
+                            <item>{{ dns_entry }}</item>
+                        {% endfor %}
+                    </dnsEntrySet>
+                {% endif %}
+                {% if vpc_end_point.private_dns_enabled  %}
+                    <privateDnsEnabled>{{ vpc_end_point.private_dns_enabled }}</privateDnsEnabled>
+                {% endif %}
+                <requesterManaged>true</requesterManaged>
+                <rreationTimestamp>{{ vpc_end_point.created_at }}</rreationTimestamp>
+                <ownerId>12345678912</ownerId>
+            {% endfor %}
+        </item>
+    </vpcEndpointSet>
+</DescribeVpcEndpointsResponse>"""
+
+DELETE_VPC_ENDPOINT_RESPONSE = """<DeleteVpcEndpointsResponse xmlns="http://ec2.amazonaws.com/doc/2016-11-15/">
+    <unsuccessful/>
```

Comment:
Can we add a TODO comment here? From the docs:
`Information about the VPC endpoints that were not successfully deleted.`

Don't mind that this isn't implemented yet, but the wording makes it a bit confusing atm - my initial reading was that the entire deletion  was unsuccessful.